DigitalLearningSolutions.Web/ViewModels/Common/ViewComponents/RadiosListItemViewModel.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace DigitalLearningSolutions.Web.ViewModels.Common.ViewComponents
 {
-    using DigitalLearningSolutions.Data.Enums;
+    using global::DigitalLearningSolutions.Data.Enums;

```

Comment:
Think this is a knock-on effect from the weird namespaces. This file shouldn't have needed to be changed.